### PR TITLE
Data augmentation

### PR DESCRIPTION
**Pull request by DianLiI**

### CODE REVIEW

File: examples/nni_data_augmentation/README.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+## Tuun data augmentation
+##### Installation
+###### This example
```
Comment (willieneis) [inline examples/nni_data_augmentation/README.md:3]: - Could we be explicit here on which directory should be in at start of installation?
- Do we have a setup.py in this directory (seems to be called below)?

File: examples/nni_data_augmentation/README.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+```shell
+conda create -n augmentation python=3.6 pip -y
```
Comment (willieneis) [inline examples/nni_data_augmentation/README.md:5]: Are we primarily/only using conda here for pytorch installation?

File: examples/nni_data_augmentation/README.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
+## Tuun data augmentation
```
Comment (willieneis) [inline examples/nni_data_augmentation/README.md:1]: Two broad things that might be nice to add to readme:
(1) Just a brief (couple-sentence) overview of what this demo carries out.
(2) Reference/link to basenet implementation we use.

**Review by willieneis — COMMENTED** [review]
Generally looks good. Some overview comments in README.

File: examples/nni_data_augmentation/README.md (hunk 1 above)
Comment (DianLiI) [reply examples/nni_data_augmentation/README.md]: Added "First cd into this folder" to the beginning of the instructions and deleted setup.py usage

File: examples/nni_data_augmentation/README.md (hunk 2 above)
Comment (DianLiI) [reply examples/nni_data_augmentation/README.md:7]: Added python venv as alternative.

File: examples/nni_data_augmentation/README.md (hunk 3 above)
Comment (DianLiI) [reply examples/nni_data_augmentation/README.md]: Added introduction at the beginning of README

**Review by willieneis — APPROVED** [review]

**Comment by willieneis** [comment]
Updates look good.

**Pull request merged**